edge/services/helloworld/README.md

```diff
@@ -176,180 +205,53 @@ hzn exchange service listpolicy <published-helloworld-service-name>
+  ```
 
-3. If you have the HZN_ORG_ID environment variable set from previous work, unset it (in a moment this value will now come from `horizon/hzn.json`):
-```
-unset HZN_ORG_ID
-```
-4. Cd to the directory in which you want to create your new service and then:
-```
-hzn dev service new -o <org-id> -s <service-name> -i <docker-image-base>
-# E.g.:  hzn dev service new -o bp@someemail.com -s bp.helloworld -i brucemp/bp.helloworld
-```
-
-5. As part of the above section "Using the Hello World Example Edge Service", you created your Exchange user credentials and edge node credentials. Ensure they are set and verify them:
-```
-export HZN_EXCHANGE_USER_AUTH="iamapikey:PUT-YOUR-API-KEY-HERE"
-hzn exchange user list
-export HZN_EXCHANGE_NODE_AUTH="PUT-ANY-NODE-ID-HERE:PUT-ANY-NODE-TOKEN-HERE"
-hzn exchange node confirm
-```
-
-### Building and Publishing Your Own Version of the Hello World Example Edge Service
-
-1. Edit `service.sh`, for example changing "Hello" to "Hey there"
-    - Note: this service is a shell script simply for brevity, but you can write your service in any language.
-2. Build the hello world docker image:
-```
-make
-```
-3. Test the service by having Horizon start it locally:
-```
-hzn dev service start -S
-```
-
-4. Check that the container is running:
-```
-sudo docker ps 
-```
-
-5. See the docker container running and look at the output:
-
-	on **Linux**:
-	```
-	tail -f /var/log/syslog | grep helloworld[[]
-	```
-
-	on **Mac**:
-	```
-	docker logs -f $(docker ps -q --filter name=helloworld)
-	``` 
-
-6. See the environment variables Horizon passes into your service container:
-```
-docker inspect $(docker ps -q --filter name=helloworld) | jq '.[0].Config.Env'
-```
-7. Stop the service:
-```
-hzn dev service stop
-```
-8. Create a service signing key pair in `~/.hzn/keys/` (if you haven't already done so):
-```
-hzn key create <my-company> <my-email>
-```
-9. Have Horizon push your docker image to your registry and publish your service in the Horizon Exchange and see it there:
-```
-hzn exchange service publish -f horizon/service.definition.json
-hzn exchange service list
-```
-10. Publish your edge node deployment pattern in the Horizon Exchange and see it there:
-```
-hzn exchange pattern publish -f horizon/pattern.json
-hzn exchange pattern list
-```
-11. Register your edge node with Horizon to use your deployment pattern (substitute for `SERVICE_NAME` the value you specified above for `hzn dev service new -s`):
-```
-hzn register -p pattern-SERVICE_NAME-$(hzn architecture)
-```
-
-12. The edge device will make an agreement with one of the Horizon agreement bots (this typically takes about 15 seconds). Repeatedly query the agreements of this device until the `agreement_finalized_time` and `agreement_execution_start_time` fields are filled in:
-```
-hzn agreement list
-```
-
-13. Once the agreement is made, list the docker container edge service that has been started as a result:
```

Comment:
Once s/b After. This is a global cmt when the word is used in this context.

Reply:
notice those are lines shaded red, which means they are being deleted in this PR

Reply:
OK